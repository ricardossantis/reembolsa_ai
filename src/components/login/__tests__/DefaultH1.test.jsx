import DefaultH1 from "../default-h1";
import React from "react";
import renderer from "react-test-renderer";

describe("Render Test", () => {
  it("renders DefaultH1", () => {
    const component = renderer.create(<DefaultH1/>).toJSON();
    expect(component).toMatchSnapshot();
  });
});

