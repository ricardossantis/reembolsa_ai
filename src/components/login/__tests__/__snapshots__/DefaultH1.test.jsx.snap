// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Test renders DefaultForm 1`] = `
<h1
  className="sc-bdnylx gispQU"
/>
`;

exports[`Render Test renders DefaultH1 1`] = `
<h1
  className="sc-bdnylx gispQU"
/>
`;
